examples/tutorials/nb_python/ECCV_2020_Navigation.py

```diff
@@ -45,7 +45,22 @@
 # @title Colab Setup and Imports { display-mode: "form" }
 # @markdown (double click to see the code)
 
-import os, sys
+import math
```

Comment:
Weird, why didn't the ipynb file update. Pre-commit runs automatically when you commit, right @bigbike ?

Reply:
I have no idea at this point. 

I even upgraded my git to the latest version. 
If I changed the .py file, ipynb will not be updated.